lib/date.js

```diff
@@ -0,0 +1,9 @@
+import { format } from 'date-fns';
+
+export function getCurrentYear() {
```

Comment:
if you wanna do arrow func
```suggestion
export const getCurrentYear = () => new Date().getFullYear();
```